doc/install_centos6.txt

```diff
@@ -16,44 +16,16 @@ Easy Installation of an optimized Theano on CentOS 6
     investing money (i.e. buying a license to a proprietary BLAS
     implementation), it is possible to gain further performance.
```

Comment:
(It's free only for academics....which I realize is what most of Theano users are anyway.) 

We tried to avoid to refer to conda whenever possible. Maybe we should rethink this, but we tried to keep it as straighforward as possible and conda makes it very complicated because is system-agnostic (up to a certain level). The idea was to provide each user with one, unique, consecutive set of instructions. Still, it might be worth to add a reference to conda in the Generic installation instructions page and link it from here. Something along these lines might work:
"If you cannot modify the python environment in your system or prefer to have a separate environment for Theano, you can check out [conda](https://www.continuum.io/downloads), which comes with the most common python packages already installed as well as an mkl blas library which is free for academic users"